### ISSUE TITLE
RowDefinition sizes and Border.Shadow don't appear to be adhered to on iOS

### ISSUE THREAD

**Issue by faceoffers28** [cemahseri/MauiFinance#2]
I'm running Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.3.6 with Xamarin.iOS version 16.0.0.75. My target is net6.0-ios14.5. Minimum target is 13.0,

Is there an easy fix for this?

**Comment by faceoffers28** [comment]
More info on the issue. https://github.com/dotnet/maui/issues/7015 Can you comment on this?

**Comment by cemahseri** [comment]
That issue is locked by [msftbot](https://github.com/apps/msftbot) as seen in https://github.com/dotnet/maui/issues/7015#event-7224125666, after it's getting fixed as seen in https://github.com/dotnet/maui/issues/7015#ref-pullrequest-1258115029 and getting closed afterwards.

Well, since Border is a new control in .NET MAUI and Microsoft encouraged us to use it, I've used it instead of Frame control. As I've mentioned in the other issue, I don't have a Mac to compile this project and test it on iOS. So I can't reproduce the bug. All I can recommend you is updating your Visual Studio, .NET, and .NET MAUI. Then download and test this repro project to see if the bug is still there: https://github.com/dotnet/maui/issues/7015#issuecomment-1123309879

If you still have that annoying bug, just uninstall your Visual Studio along with other dependencies and do a fresh install and be entirely sure that it's not related to Visual Studio or you. If this bitch ass bug still persists, go ahead and create an issue on .NET MAUI repo and mention the issue you've mentioned here. Don't forget to post a minimal repro project/code in the issue you are going to create.



**TL;DR:** I haven't faced this issue since I haven't tested this on iOS. Make a clean install of latest Visual Studio, test that repro project. If the bug is there, create an issue on the .NET MAUI repo and mention [this](https://github.com/dotnet/maui/issues/7015) issue and post the repro code.
